Java/JDI/jdi-cucumber-test-generator/src/main/webapp/WEB-INF/static/js/case/filterCases.js

```diff
@@ -0,0 +1,19 @@
+function filterCases() {
+    var isAllChecked = $('input:checkbox:checked.checkedAllTags').map(
```

Comment:
Maybe replace 'input:checkbox:checked.checkedAllTags' to '.checkedAllTags:checked'